helm-service/pkg/helm/helm_helper.go

```diff
@@ -66,13 +70,40 @@ func GetChartName(service string, generated bool) string {
 	return service + suffix
 }
 
-// GetReleaseName returns the name of the Helm release
+// GetReleaseName returns the name of the Helm release, e.g., ${PROJECT}-${STAGE}-${SERVICE}[-generated]
 func GetReleaseName(project string, stage string, service string, generated bool) string {
 	suffix := ""
 	if generated {
 		suffix = "-generated"
 	}
-	return project + "-" + stage + "-" + service + suffix
+
+	proposedReleaseName := project + "-" + stage + "-" + service + suffix
```

Comment:
Imho this would be easier to read than nesting those `if`s: 

```
if len(proposedReleaseName) <= serviceNameMaxLen {
  return proposedReleaseName
}

proposedReleaseName = service + suffix
if len(proposedReleaseName) <= serviceNameMaxLen {
  return proposedReleaseName
}
// ... you get the idea ;)
```

Reply:
Agreed, will adapt